game_util/scene_items.py

```diff
@@ -185,6 +187,33 @@ def handle_event(self, event, item_loc, is_rock_dirty) :
                     self.interacting_pet.set_location(self.item_location[0],self.item_location[1]+ self.item_rect.height/2.5)
                     self.interacting_pet.set_current_animation(config.RockActions.sleeping.value, True)
 
+class Score:
+    def __init__(self, pygame, screen):
+        self.font = pygame.font.Font(None,36)
+        self.pygame = pygame
+        self.screen  = screen
+        self.score = 0
```

Comment:
can you change this variable int `score_value` instead of `score`, its just because the class is score and having the actual value be something different would be more readable.

Reply:
Sure, I will change it.